src/components/settings/SettingsLMSTab/LMSSelectorPage.tsx

```diff
@@ -38,6 +38,7 @@ export function LMSSelectorPage(setLms: (string) => void) {
             onChange={handleChange}
             name="colors"
             columns={3}
+            ariaLabelledby="lms-selection-heading"
```

Comment:
The `<p>` above with the text "Select the LMS or LXP you want to integrate..." is a more useful label for the `SelectableBox.Set` than "Let's get started"... 

"Let's get started" doesn't really describe the choices in the `SelectableBox.Set`.

Reply:
@BilalQamar95 Bump on this comment. Can we move the `id="lms-selection-heading"` to the `<p>` instead so it's a more useful/semantically correct message?

```
<p id="lms-selection-heading">Select the LMS or LXP you want to integrate with edX For Business.</p>
```